docker/nginx.conf

```diff
@@ -4,11 +4,15 @@ server {
     location / {
         root   /usr/share/nginx/html;
         index  index.html;
-        try_files $uri /index.html;                 
+        try_files $uri /index.html;
 
         location = /config/env-config.js {
             add_header Cache-Control "no-store";
         }
+
+        location = /index.html {
```

Comment:
Why do we need to add this? Our CDN shouldn't be caching a .html file. Should we add no-store to everything?